Libraries/xcsdk/Sources/SDK/Manager.cpp

```diff
@@ -47,7 +47,9 @@ findTarget(std::string const &name) const
 
         /* If the platform name matches but no targets do, use any target. */
         if (platform->name() == name || platform->path() == name) {
-            return platform->targets().back();
+            if (!platform->targets().empty()) {
+                return platform->targets().back();
```

Comment:
Honestly it seems like an error case to have a platform with no SDKs.

Avoid returning a default SDK for a platform with no SDKs: LGTM